dpc-go/dpc-api/src/v2/ssas_controller.go

```diff
@@ -28,6 +29,107 @@ func NewSSASController(ssasClient client.SsasClient, attrClient client.Client) *
 	}
 }
 
+// DeleteKey function to delete a public key from ssas system
+func (sc *SSASController) DeleteKey(w http.ResponseWriter, r *http.Request) {
+	log := logger.WithContext(r.Context())
+	implementorID, _ := r.Context().Value(middleware.ContextKeyImplementer).(string)
```

Comment:
Here and throughout. 🤓
```suggestion
	implementerID, _ := r.Context().Value(middleware.ContextKeyImplementer).(string)
```